benchmark/wikisql/wiki_sql.py

```diff
@@ -102,11 +103,16 @@ def get_tables_and_questions(source_file):
             sqls.append(item['sql'])
     return tables, questions, results, sqls
 
+def similarity(spark_ai_result, expected_result):
```

Comment:
We should use the similarity in the transform_df method and ask gpt to pick to most relevant column value.

Reply:
will make a follow up PR